src/main/java/com/teamacronymcoders/epos/utils/helpers/PlayerHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.teamacronymcoders.epos.utils.helpers;
+
+import net.minecraft.entity.player.PlayerEntity;
+
+public class PlayerHelper {
+    public static boolean eventChecks(PlayerEntity playerEntity) {
+        return isNotNull(playerEntity) && !isCreative(playerEntity) && !isSpectator(playerEntity);
+    }
+
+    private static boolean isNotNull(PlayerEntity playerEntity) {
```

Comment:
These isNotNull, isCreative, and isSpectator checks are kind of pointless given calling the check via this helper class is longer than just directly querying the player entity.